### PR TITLE
Accounts DB fixes + prints

### PR DESCRIPTION
**Pull request by iulianpascalau**
- removed some direct nil comparison in accounts DB
- added prints for IsValidator function

### CODE REVIEW

**Review by SebastianMarian — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by LucianMincu — DISMISSED** [review]
👍

**Review by SebastianMarian — DISMISSED** [review]

File: integrationTests/vm/systemVM/stakingSC_test.go [hunk 1]
```diff
@@ -200,6 +219,25 @@ func TestStakingUnstakingAndUnboundingOnMultiShardEnvironmentWithValidatorStatis
 
 	checkAccountsAfterStaking(t, nodes)
 
+	//manually setting every un-staked node status to inactive
```
Comment (sasurobert) [inline integrationTests/vm/systemVM/stakingSC_test.go:222]: Take out in a function. Duplicated code

**Review by sasurobert — CHANGES_REQUESTED** [review]

**Review by SebastianMarian — APPROVED** [review]

File: integrationTests/vm/systemVM/stakingSC_test.go (hunk 1 above)
Comment (iulianpascalau) [reply integrationTests/vm/systemVM/stakingSC_test.go]: done.

**Review by sasurobert — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
👍

**Pull request merged**